Server/ConcordiaCurriculumManager/Services/DossierReviewService.cs

```diff
@@ -46,6 +54,11 @@ public async Task SubmitDossierForReview(DossierSubmissionDTO dto)
         if (!(await _groupService.IsGroupIdListValid(dto.GroupIds))) throw new InvalidInputException("All groups set as reviewers must be valid groups");
 
         Dossier dossier = await _dossierService.GetDossierDetailsByIdOrThrow(dto.DossierId);
+        dossier.Discussion = new()
```

Comment:
Can we just create the discussion board with the initial dossier creation, that way we don't have to worry about nulls?